tests/components/surepetcare/__init__.py

```diff
@@ -72,8 +72,5 @@
     DOMAIN: {
         CONF_USERNAME: "test-username",
         CONF_PASSWORD: "test-password",
-        "feeders": [12345],
```

Comment:
Shouldn't we keep these items until we invalidate them in the config? At the moment they are just deprecated.